### ISSUE TITLE
modprobe'ing loop device inside container fails

### ISSUE THREAD

**Issue by eddyg** [ltsp/ltsp#70]
I'm running `Ubuntu 18.04.3 LTS`, `lxc 3.0.3`, with `ltsp 19.11-1~201911110854~ubuntu18.04.1` installed *in* an lxc container, which allows me to run the `ltsp` commands needed to generate the config files and images.

When I run `ltsp image` (and more specifically, the `ltsp kernel` step that occurs after the `squashfs` is generated), it stops with the error:
```
modprobe: ERROR: ../libkmod/libkmod.c:586 kmod_search_moddep() could not open moddep file '/lib/modules/5.0.0-32-generic/modules.dep.bin'
modprobe: FATAL: Module loop not found in directory /lib/modules/5.0.0-32-generic
LTSP command failed: modprobe loop max_part=9
```
Inside a container, there is no `/lib/modules` at all, but *if* the loop devices have been properly configured in the container config (see [this gist](https://gist.github.com/kaxing/f90e66a1041c8686d360a480b15be8fe) for reference), then *not* calling `modprobe loop max_part=9` seems to allow things to work.

In other words, when I comment out this line, and even though I'm using an lxc as a way to generate my LTSP images and configs, `ltsp kernel` is able to mount the squashfs image it just created and extract the `initrd.img` and `vmlinuz` files.

**Comment by alkisg** [comment]
Thank you Eddy, I will add an `if [ -d /sys/modules/loop ]` to avoid modprobe when it's already loaded, as it is in Ubuntu (it's build-in).

**Comment by alkisg** [comment]
Fix committed in https://github.com/ltsp/ltsp/commit/2689a204aad1d7268d5ac723979e14b715209778.
